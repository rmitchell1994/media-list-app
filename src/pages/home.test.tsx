import React from "react";
import Home from "./home";
import { render } from "@testing-library/react";

describe("Home", () => {
  it("should render default compon", () => {
    const { asFragment } = render(<Home />);

    expect(asFragment()).toMatchSnapshot();
  });
});
