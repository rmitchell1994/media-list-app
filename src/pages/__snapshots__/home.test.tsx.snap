// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render default component 1`] = `
<DocumentFragment>
  <div>
    Home
  </div>
</DocumentFragment>
`;
